backend/src/server.cpp

```diff
@@ -1,4 +1,21 @@
 #include "server.hpp"
+#include "cryptography.hpp"
+
+namespace
+{
+    std::vector<std::string> encodePasswords(const std::vector<std::string> &passwords)
+    {
+        std::vector<std::string> result;
+        result.reserve(passwords.size());
+
+        for (const auto &password : passwords)
+        {
+            result.push_back(crow::utility::base64encode(password, password.size()));
+        }
+
+        return result;
+    }
+}
```

Comment:
**thought:** I'm wondering if we should encode+encrypt in the same function later. It's nice for separation right now but it could help optimize